framework/src/postprocessors/NumFailedTimeSteps.C

```diff
@@ -0,0 +1,40 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "NumFailedTimeSteps.h"
+#include "Transient.h"
+
+registerMooseObject("MooseApp", NumFailedTimeSteps);
+
+InputParameters
+NumFailedTimeSteps::validParams()
+{
+  InputParameters params = GeneralPostprocessor::validParams();
+  params.addClassDescription("Collects the number of failed time steps from the time stepper.");
+  return params;
+}
+
+NumFailedTimeSteps::NumFailedTimeSteps(const InputParameters & parameters)
+  : GeneralPostprocessor(parameters), _timestepper(NULL)
```

Comment:
```suggestion
  : GeneralPostprocessor(parameters)
```
I dont think you need that